jest.config.js

```diff
@@ -1,11 +1,11 @@
-module.exports = () => ({
+module.exports = {
   clearMocks: true,
   preset: 'ts-jest',
-  testMatch: ['**/__tests__/**/*test.(js|ts)'],
+  testMatch: ['**/*test.(js|ts)'],
```

Comment:
I kinda just forced everyone into using the `__tests__` dir by adding it into the template, so I'm open to opinions outside of this pr if that's something we want to continue with. But in general I would like keep configs consistent.

Reply:
I agree we should be consistent. I don't love `__tests__`, but I supposed it to be a reflection of what the JS community favors these days. I've grown to love tests next to the files they test because `import './thing'` is so beautiful compared to `import '../src/thing'`, and they can just move around together.

Reply:
I've actually started to dislike writing tests next to the files they test. Logically, tests are separate from implementation, and I got really tired of running `ls` and seeing double everywhere. The `__` naming convention is a little silly IMO but I'm definitely on board for the idea of putting tests in a separate directory.

As for the imports, it's just `import '../thing'` instead of `import './thing'`.

Reply:
😸

Reply:
Ended up going back to this matching pattern in #2. Turns out having test files within `src/steps` messes up the way the SDK pulls files.